tests/testthat/test-OTHER-create-track-hub-nojointpeaks.R

```diff
@@ -70,3 +70,10 @@ test_that("trackDb.txt file has the correct link to the bigwig files", {
   expected.coverage.url <- paste0(url, "/samples/", bigWig.part.vec[1], "/", "coverage.bigWig")
   expect_equal(coverage.url, expected.coverage.url)
 })
+
+test_that("trackDb.txt file must not have link for joint_peaks bigwig", {
+  trackDb.vec <- readLines(trackDb.txt)
+  coverage.line <- grep("samples/kidney/MS002201/joint_peaks.bigWig", trackDb.vec)
+  coverage.url <- sub(".* ", "", trackDb.vec[coverage.line])
```

Comment:
good start. but please fix
1. remove this line (coverage.url) because it would be simpler to just test if there is a line with joint_peaks.bigWig (no need to use `sub`
2. just grep "joint_peaks.bigWig" (no need for samples/kidney/etc prefix.
3. change variable name from coverage.line to something more relevant, e.g. joint_peaks.bigWig.lines

Reply:
Okay, I'm on it